src/Common/tests/System/Collections/ISet.Generic.Tests.cs

```diff
@@ -419,6 +419,7 @@ public void ISet_Generic_ExceptWith_Itself(int setLength)
 
         [Theory]
         [MemberData(nameof(ValidCollectionSizes))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
+1. Also can we add that comment on the same line? we have been doing that to be able to check how many tests need renabling by doing a findall without the need of opening each test file.